streamPair.q

```diff
@@ -0,0 +1,85 @@
+// import linear regression module
+\l linear_regression.q
+\l kalman_filter.q
+
+// load tables
+tab1: 40000#1_ flip `dateTime`bid`ask`bidVol`askVol!("*FFFF";",") 0: `:data/USA500IDXUSD.csv;
+tab2: 40000#1_ flip `dateTime`bid`ask`bidVol`askVol!("*FFFF";",") 0: `:dataUSATECHIDXUSD.csv;
+tab3: flip `dateTime`spread`mean`up`low`operation!("P"$();"F"$();"F"$();"F"$();"F"$();"F"$());
+historial_tab1: 40000#1_ flip `open`high`low`close`adjClose`vol!("FFFFFF";",") 0: `:data/NASDAQ100_hist.csv;
+historial_tab2: 40000#1_ flip `open`high`low`close`adjClose`vol!("FFFFFF";",") 0: `:data/SP500_hist.csv;

```

Comment:
We no longer need the 40000 because the historical data should have the same length. And it doesn't matter if the real-time data have the same length.